New "Update" page (#615)
MicrosoftDocs/quantum-docs-pr: articles/install-guide/update.md

2. Select **Terminal** -> **New Terminal**
3. Follow the instructions for updating using the command line (directly below)

### Update Q# projects using the command line

1. Navigate to the folder containing your project file
2. Run the following command:
	```bash
	dotnet clean [project_name].csproj
	```

3. In each of your .csproj files, update the target framework to `netcoreapp3.0` (or `netstandard2.1` if it is a library project).
	That is, edit lines of the form:
	```xml
	<TargetFramework>netcoreapp3.0</TargetFramework>
	```
	You can find more details on specifying target frameworks [here](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks).
4. Run the following command:
	```bash
	dotnet add package Microsoft.Quantum.Development.Kit
	```

	If your project uses any other Microsoft.Quantum packages, run the command for these too.
5. Save and close all files.
6. Repeat 1-4 for each project dependency, then navigate back to the folder containing your main project and run:
	```bash
	dotnet build [project_name].csproj
	```

With your Q# projects now updated, follow the instructions below to update the QDK itself.

> ## 2. Updating the QDK

Author: ```suggestion
## Updating the QDK
```

This article assumes that you already have the QDK installed. If you are installing for the first time, then please refer to the [installation guide](xref:microsoft.quantum.install).

> If you saw that a new version of the QDK was released and you would like to get with the times, then you're in the right place.

Reviewer: ```suggestion
We recommend keeping up to date with the latest QDK release. Follow this update guide to upgrade to the most recent QDK version. 
```

The process consists of two parts:
1. updating your existing Q# files and projects to align your code with any updated syntax
2. updating the QDK itself for your chosen development environment 

> ## 1. Updating Q# Projects 

Reviewer: ```suggestion
## Updating Q# Projects 
```

Author: @bromeg this one I made numbered to align with the two "parts" of the update process that are described at the very beginning:
1. Updating Q# projects
2. Updating the QDK

Personally I think it's beneficial to keep them as numbered H2's, but just let me know what you think (if we go no-numbers, I'll remove the number from the "Update QDK" H2 as well)

Author: Also deleting the number from the second H2  👍

	```bash
	dotnet --version
	```
Verify the output is `3.0.100` or higher, then follow the instructions below depending on your setup (Visual Studio, Visual Studio Code, or directly the command line).

### Update Q# projects in Visual Studio
 
1. Update to the latest version of Visual Studio 2019, see [here](https://docs.microsoft.com/visualstudio/install/update-visual-studio?view=vs-2019) for instructions
2. Open your solution in Visual Studio
3. From the menu, select **Build** -> **Clean Solution**
4. In each of your .csproj files, update the target framework to `netcoreapp3.0` (or `netstandard2.1` if it is a library project).
	That is, edit lines of the form:
	```xml
	<TargetFramework>netcoreapp3.0</TargetFramework>
	```
	You can find more details on specifying target frameworks [here](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks).
5. Save and close all files in your solution
6. Select **Tools** -> **Command Line** -> **Developer Command Prompt**
7. For each project in the solution, run the following command:
	```bash
	dotnet add [project_name].csproj package Microsoft.Quantum.Development.Kit
	```

> 	If your projects use any other Microsoft.Quantum packages, run the command for these too.

Author: @bettinaheim @cgranade  @bromeg , could you give me an example or two of what one of these other packages might be? 
I'd like to put an "(e.g. ...)" here to avoid inexperienced users thinking this means they have to run it for namespaces (Microsoft.Quantum.Intrinsic, etc.)

Reviewer: I think this is if they are using additional libraries, so I think Microsoft.Quantum.Chemistry might be an example, but I'd like @bettinaheim  and @cgranade  to confirm

### Update Q# projects using the command line

1. Navigate to the folder containing your project file
2. Run the following command:
	```bash
	dotnet clean [project_name].csproj
	```

3. In each of your .csproj files, update the target framework to `netcoreapp3.0` (or `netstandard2.1` if it is a library project).
	That is, edit lines of the form:
	```xml
	<TargetFramework>netcoreapp3.0</TargetFramework>
	```
	You can find more details on specifying target frameworks [here](https://docs.microsoft.com/dotnet/standard/frameworks#how-to-specify-target-frameworks).
4. Run the following command:
	```bash
	dotnet add package Microsoft.Quantum.Development.Kit
	```

	If your project uses any other Microsoft.Quantum packages, run the command for these too.
5. Save and close all files.
6. Repeat 1-4 for each project dependency, then navigate back to the folder containing your main project and run:
	```bash
	dotnet build [project_name].csproj
	```

With your Q# projects now updated, follow the instructions below to update the QDK itself.

## 2. Updating the QDK

The process to update the QDK varies depending on your development language and environment.

> Select yours below to be taken to the proper instructions.

Reviewer: ```suggestion
Select your development environment below.
```

* [Python: update the IQ# extension](#update-iq-for-python)
* [Jupyter Notebooks: update the IQ# extension](#update-iq-for-jupyter-notebooks)
* [Visual Studio: update the QDK extension](#update-visual-studio-qdk-extension)
* [VS Code: update the QDK extension](#update-vs-code-qdk-extension)
* [Command-line and C#: update project templates](#c-using-the-dotnet-command-line-tool)

### Update IQ# for Python

1. Update the `iqsharp` kernel

    ```bash
    dotnet tool update -g Microsoft.Quantum.IQSharp
    dotnet iqsharp install
    ```

> 	If after the second command you see an error message `Could not install iqsharp into your Jupyter configuration...`, do not fret. 

Reviewer: ```suggestion
	If after the second command you see an error message `Could not install iqsharp into your Jupyter configuration...`, you can keep following the steps below and enter the command again after step 3. 
```

Author: Yeah I was unsure myself, just put the workaround as a patch solution. Upon now testing it again, I didn't get the error message, but I'm not sure if that means it's fixed for everyone

> 	Simply keep following the steps below, and enter the command again after step 3.

Reviewer: ```suggestion
```
